packages/transaction-log/src/services/api.service.ts

```
@@ -80,3 +81,47 @@ export async function getServerHeartBeat(): Promise<{
+  batchSize: number,
+  paused: boolean
+) {
+  try {
+    const payload = {
+      tids: transactions.map(t => t._id),
+      batchSize,
+      paused
+    }
+    await addToTaskQueue(payload)
+  } catch (error) {
+    throw error
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)